howso/client/base.py

```diff
@@ -645,10 +651,17 @@ def train(  # noqa: C901
                     "series": series,
                     "session": self.active_session.id,
                     "skip_auto_analyze": skip_auto_analyze,
+                    "skip_reduce_data": skip_reduce_data,
                     "train_weights_only": train_weights_only,
                 })
+
                 if response and response.get('status') == 'analyze':
-                    needs_analyze = True
+                    status['status'] = 'analyze'
+                    status['details'] = 'An analyze is strongly recommended for this trainee.'
+                elif response and response.get('status') == 'reduce_data':
+                    status['status'] = 'reduce_data'
+                    status['details'] = 'Data reduction via `reduce_data` is recommended for this trainee.'
```

Comment:
I am aware that the return value of the base client's `train` is swallowed up by `Trainee.train` and used to set the `needs_analyze` property of the Trainee. However, I cannot find any instances of `needs_analyze` being accessed. Is `needs_analyze` only there for user convenience if they choose? Would it make sense to display the proposed `details` here as a warning instead? Otherwise, maybe the return type of the base class is better left as just a string to indicate the status for internal use.

Reply:
You understand correctly. I think this is mirrored in the platform client as well so the same change might be needed there? Not exactly sure off the top of my head.